src/index.html

```diff
@@ -493,5 +493,22 @@ <h2>What's that data type?</h2>
       </div>
     </div>
 
+    <div class="exercise">
+      <div class="title">
+        <h2>Python in DataCamp Light 3</h2>
+        <p>In the html code for this exercise you can specify the packages you want to use.</p>
+      </div>
+      <div data-datacamp-exercise data-lang="python" packages="pandas==0.24.1, alabaster==0.7.12" data-height="auto">
```

Comment:
About the API, I was thinking to have a structure to list packages, something like:

```html
<ul data-type="packages">
  <li data-name="pandas" data-version="0.24.1"></li>
  <li data-name="alabaster" data-version="0.7.12"></li>
</ul>
```

Reply:
HTML has a semantic meaning to describe your document structure. I would prefer to not abuse it to shoehorn in our data model. It's a fair point that the current implementation is a bit cumbersome to write. We could move it into a div or come up with a different syntax. But for now I would leave it as a data attribute which is simple and still easily allows us to change this in the future.